packages/lib-core/package.json

```diff
@@ -28,24 +28,14 @@
     "prepublishOnly": "yarn test",
     "build": "rm -rf dist && yarn run -T rollup -c && yarn process-dts",
     "lint": "yarn run -T eslint $INIT_CWD",
-    "test": "yarn run -T test $INIT_CWD --passWithNoTests",
+    "test": "yarn run -T test $INIT_CWD",
     "process-dts": "yarn run -T downlevel-dts dist/index.d.ts dist/index.ts38.d.ts --to=3.8"
   },
   "peerDependencies": {
-    "react": "^17.0.2",
-    "react-router": "^5.2.1",
-    "redux": "^4.1.2"
+    "react": "^17.0.2"
   },
   "dependencies": {
     "lodash-es": "^4.17.21",
     "yup": "^0.32.11"
-  },
-  "peerDependenciesMeta": {
```

Comment:
Did we not actually use react-router/redux in the core package? Not sure how these got in here.

Reply:
Ah - I think I get it. These were for the extensions we moved. Nvm.

Reply:
Correct, these were only used by extension types, which got moved into a separate package.

They were marked as optional via `peerDependenciesMeta` because you could use the core package in your host application or plugins without using any/all of the extension types.